assets/dev/scss/editor/_global.scss

```diff
@@ -54,6 +54,19 @@
 	.elementor-button {
 		margin-block-start: 20px;
 	}
+
+	&--upsale {
+		padding: 15px 30px 30px;
+
+		.elementor-nerd-box-title {
+			font-weight: 500;
+			line-height: 1.2;
+		}
+
+		.elementor-nerd-box-message {
+			line-height: 1.5;
+		}
```

Comment:
Not a blocker. This can be nested:

```
.elementor-nerd-box {
    &-title {
	font-weight: 500;
	line-height: 1.2;
    }
    
    &-message {
	line-height: 1.5;
    }
}
```

Reply:
I don't think this will work unless we start overloading the dom with more classes.

i.e. `<div class="elementor-nerd-box-title elementor-nerd-box--upsale-title">{{ elementData.upsale_data.title }}</div>`